{"uuid":"278d79e2-1f33-46ed-a0ca-1be4ad711212","historyId":"22d46786b87e946bc0b8338635d02598","fullName":"testCases.TestCases1.test_2","labels":[{"name":"package","value":"testCases.TestCases1"},{"name":"testClass","value":"testCases.TestCases1"},{"name":"testMethod","value":"test_2"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"C:/Users/AbdElRahman/IdeaProjects/TestAutomationExercise-Website"},{"name":"subSuite","value":"testCases.TestCases1"},{"name":"host","value":"DESKTOP-9QSEPRP"},{"name":"thread","value":"33212@DESKTOP-9QSEPRP.TestNG-test-C:/Users/AbdElRahman/IdeaProjects/TestAutomationExercise-Website-1(20)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"epic","value":"EP001"},{"name":"feature","value":"Login"}],"links":[],"name":"test_2","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Expected a ',' or '}' at 2107 [character 5 line 85]","trace":"org.json.JSONException: Expected a ',' or '}' at 2107 [character 5 line 85]\r\n\tat org.json.JSONTokener.syntaxError(JSONTokener.java:503)\r\n\tat org.json.JSONObject.<init>(JSONObject.java:264)\r\n\tat org.json.JSONTokener.nextValue(JSONTokener.java:409)\r\n\tat org.json.JSONArray.<init>(JSONArray.java:105)\r\n\tat org.json.JSONArray.<init>(JSONArray.java:142)\r\n\tat utils.Utilities.retrieveAllTestCases(Utilities.java:74)\r\n\tat utils.Utilities.getTestCase(Utilities.java:88)\r\n\tat testCases.TestCases1.test_2(TestCases1.java:40)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base/java.lang.Thread.run(Thread.java:833)\r\n"},"stage":"finished","description":"Test Case 2: Login User with correct email and password","steps":[],"attachments":[],"parameters":[],"start":1713052519698,"stop":1713052519699}